### ISSUE TITLE
Feature Request: Configureable keybindings

### ISSUE THREAD

**Issue by N-R-K** [ajbura/cinny#171]
#### Is your feature request related to a problem? Please describe.

Currently keyboard support feels a bit lackluster. There's a couple open issues which IMO are a step in the right direction (#132 #97 #57). But the main problem is most keyboard users have their own preference in keybindings. For example I dislike having to reach the arrow key as it's further away from the home row.

#### Describe the solution you'd like

Making keybindings customizable would mean users can pick the keybinding they're comfortable without needing to put up with default keybindings which they may not like.

Requesting this feature sooner rather than later since I assume it would be less work to build new keybindings with goal of them being customizable compared to hard-coding them and then having to make them configurable later on.
